<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>285</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>285</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>285</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>14</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Matrix Choice</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mtxChoiceComboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>11</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>NONE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MTX2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MTX3</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="resetPushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>12</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>259</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>IPAddress</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipAddressLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>259</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="connectLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>NOT CONNECTED</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>470</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>248</y>
     <width>470</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>60</y>
     <width>431</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ProcAmp Gain</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSlider" name="procAmpGainSlider">
      <property name="maximum">
       <number>150</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="procAmpGainValue">
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string> 1.000</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ProcAmp Black</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSlider" name="procAmpBlackSlider">
      <property name="minimum">
       <number>-200</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="procAmpBlackValue">
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string> 1.000</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ProcAmp Hue</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QSlider" name="procAmpHueSlider">
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="procAmpHueValue">
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string> 1.000</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ProcAmp Saturation</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QSlider" name="procAmpSaturationSlider">
      <property name="maximum">
       <number>150</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="procAmpSatValue">
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string> 1.000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
